dxchange/reader.py

```diff
@@ -661,13 +661,40 @@ def read_dx_meta(file_name, label1='/measurement/', label2='/process/'):
     return meta
 
 
-def create_standard_meta(file_name):
-    pass
+def create_standard_dx_meta(file_name):
+    """
+    Create meta data dict for files written in standard dxfile format
+
+    :param file_name: file name of the HDF5 input file that follows dxfile standard file structure
+    :return meta: meta data dictionary using standard names for keys
+    """
+
```

Comment:
TODO: add versioning

Reply:
The basic meta data for mosaic are the file names and the sample x-y position at each scan so one solution could be to add sample x-y in the create_standard_meta(), then I can add a read_mosaic() that returns a mosaic file name grid:
```
             y_0                                 y_1                         y_2                         y_3                    y_4
x_0  2021-09/mosaic_test_036.h5  2021-09/mosaic_test_037.h5  2021-09/mosaic_test_038.h5  2021-09/mosaic_test_039.h5  2021-09//mosaic_test_040.h5
x_1  2021-09/mosaic_test_041.h5  2021-09/mosaic_test_042.h5  2021-09/mosaic_test_043.h5  2021-09/mosaic_test_044.h5  2021-09/mosaic_test_045.h5
x_2  2021-09/mosaic_test_046.h5  2021-09/mosaic_test_047.h5  2021-09/mosaic_test_048.h5  2021-09/mosaic_test_049.h5  2021-09/mosaic_test_050.h5
```